engine-precompiles/src/xcc.rs

```diff
@@ -42,7 +42,8 @@ pub mod costs {
     /// `NEAR Gas / EVM Gas`, we simply multiply `0.175 * 10^12 / 10^3 = 175 * 10^6`.
     pub const CROSS_CONTRACT_CALL_NEAR_GAS: u64 = 175_000_000;
 
-    pub const ROUTER_EXEC: NearGas = NearGas::new(7_000_000_000_000);
+    pub const ROUTER_EXEC_BASE: NearGas = NearGas::new(7_000_000_000_000);
+    pub const ROUTER_EXEC_PER_CALLBACK: NearGas = NearGas::new(12_000_000_000_000);
```

Comment:
How are these numbers determined? As I understand this is just for scheduling new transactions, but the cost seems higher than expected since creating new promises should be on the order of 1TGas.